prawdditions/__init__.py

```diff
@@ -1,6 +1,16 @@
 from .const import __version__
 from praw import Reddit, const
 
+"""
```

Comment:
In python its common for the doc strings to be contained within the method itself. See here for example:

https://github.com/praw-dev/praw/blob/b0ec83f8237bb8541201d7e0ba52b7e389ed5392/praw/reddit.py#L24

Also you can run the `pydocstyle` program if you want to get some information on formatting. I also recommend `pycodestyle` for code formatting.